zdm/energetics.py

```diff
@@ -186,6 +217,47 @@ def vector_cum_gamma_spline(Eth:np.ndarray, *params):
     result[low]=1.
     return result
 
+def vector_cum_gamma_linear(Eth:np.ndarray, *params):
+    """ Calculate cumulative Gamma function using a linear interp1d
+
+    Args:
+        Eth (np.ndarray): [description]
```

Comment:
Eth: detection threshold energy in ergs
Return: cumulative probability above Eth

Reply:
added